### PR TITLE
Agent configurations

### PR DESCRIPTION
**Pull request by guytamir10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/MicrosoftADFS/AgentConfigs/MicrosoftADFS/MicrosoftADFS.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+winlogbeat.event_logs:
+  - name: Security
+    event_id: 510, 1200, 1201, 1202, 1203, 1204, 1205, 1206, 1207
+  - name: AD FS/Admin
+  processors:
```
Comment (yaakovi) [inline Packs/MicrosoftADFS/AgentConfigs/MicrosoftADFS/MicrosoftADFS.yml:5]:
```suggestion
    processors:
```

Comment (guytamir10) [reply Packs/MicrosoftADFS/AgentConfigs/MicrosoftADFS/MicrosoftADFS.yml:5]: thanks

File: Packs/MicrosoftWSUS/AgentConfigs/MicrosoftWSUS/MicrosoftWSUS.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+filebeat.modules:
+  module: iis
```
Comment (evisochek) [inline Packs/MicrosoftWSUS/AgentConfigs/MicrosoftWSUS/MicrosoftWSUS.yml:2]: Do WSUS need to use IIS module?

Comment (evisochek) [reply Packs/MicrosoftWSUS/AgentConfigs/MicrosoftWSUS/MicrosoftWSUS.yml:2]:
```
filebeat.inputs:
- type: filestream
  enabled: true
  paths:
    - C:\Program Files\Update Services\LogFiles\Change.log
    - C:\Program Files\Update Services\LogFiles\SoftwareDistribution.log
  processors:
    - add_fields:
        fields:
          vendor: msft
          product: WSUS
```

**Review by evisochek — CHANGES_REQUESTED** [review]

File: Packs/Tableau/AgentConfigs/Tableau/Tableau.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+type: filestream
+    enabled: true
```
Comment (limorf) [inline Packs/Tableau/AgentConfigs/Tableau/Tableau.yml:2]: invalid yaml

**Review by bziser — APPROVED** [review]

**Review by evisochek — APPROVED** [review]

**Pull request merged**